darts/models/forecasting/forecasting_model.py

```diff
@@ -1475,19 +1514,16 @@ def predict(
 
         super().predict(n, num_samples)
 
-        if self._expect_covariate and future_covariates is None:
-            raise_log(
-                ValueError(
-                    "The model has been trained with `future_covariates` variable. Some matching "
-                    "`future_covariates` variables have to be provided to `predict()`."
+        # ignore TransferableFutureCovariatesLocalForecastingModel, as already applied
+        if not isinstance(self, TransferableFutureCovariatesLocalForecastingModel):
```

Comment:
(nitpicking): It'd be better if we find more OO-compliant ways to avoid such type checks (e.g., overwritting a method in `TransferableFutureCovariatesLocalForecastingModel`)